cmd/storage-rest-client.go

```diff
@@ -234,7 +248,7 @@ func (client *storageRESTClient) CreateFile(volume, path string, length int64, r
 	values.Set(storageRESTVolume, volume)
 	values.Set(storageRESTFilePath, path)
 	values.Set(storageRESTLength, strconv.Itoa(int(length)))
-	respBody, err := client.call(storageRESTMethodCreateFile, values, r, length)
+	respBody, err := client.call(storageRESTMethodCreateFile, values, ioutil.NopCloser(r), length)
```

Comment:
http client was calling Close() on the reader (whenever inter node network broke) because of which writes on the Pipe (from bitrot-streaming) was getting ErrClosePipe. Instead the writes should see the error returned by `client.call` which will happen with this change.